CWE-94-Python_darwin/greed2411_Flask-YouTube/ydl.py

```
@@ -0,0 +1,103 @@
+	Playlist of audio files,
+	Playlist of video files
+	from a URL belonging to https://www.youtube.com/
+
+	Started working on : 21-06-2017
+
+"""
+
+
+def fetch_name(url):
+    """ To get the title of the YouTube Page """
+    youtube_page = requests.get(url)
```

Ruff (0.13.2)

Probable use of `requests` call without timeout

(S113)